OpenAI-DotNet/Images/ImageGenerationRequest.cs

```diff
@@ -66,5 +66,11 @@ public ImageGenerationRequest(string prompt, int numberOfResults = 1, ImageSize
         /// </summary>
         [JsonPropertyName("user")]
         public string User { get; }
+
+        /// <summary>
+        /// Optional: The format in which the generated images are returned. Must be one of url or b64_json. Defaults to url
```

Comment:
So the ImageResult.cs expects a url. we need to update the endpoint to handle the new deserialized format.

Reply:
Ok, then I guess it's a breaking change (major bump)? I was using version 6.3.1 and it was returning a string. Was there a breaking change on a minor bump making it an URI?

Reply:
Nah, I don't think it's a breaking change if you do it right.

Reply:
> Was there a breaking change on a minor bump making it an URI?

I don't understand the question

Reply:
> Ok, then I guess it's a breaking change (major bump)?

This shouldn't be a major or minor if you do it right.

Reply:
Ah, right. Agreed. The API returns a _different_ property. The docs did not mention it, but I just tested it now. I'll add the new property `b64_json` to ImageResult.